modules/mono/glue/cs_files/Basis.cs

```diff
@@ -350,7 +350,7 @@ public Vector3 xform_inv(Vector3 v)
             );
         }
 
-		public Quat Quat() {
+		public Quat get_quat() {
```

Comment:
Should be `public static implicit operator Quat(Basis from)` instead.

Reply:
Maybe you can add those in the next round because I'm really not that familiar with C#.

However, just like get/set for euler, scale and axis_angle, set_quat/get_quat should also be there. They were just missing from the API.

Reply:
Good, I have some modifications to Basis to push so I'll change it.